src/components/layout/dt-tile/dt-tile.js

```diff
@@ -91,6 +89,8 @@ export class DtTile extends DtBase {
       title: { type: String },
       expands: { type: Boolean },
       collapsed: { type: Boolean },
+      rowgap: { type: String },
+      columngap: { type: String },
```

Comment:
Like the above comment about the CSS property, let's define just a single `gap` property since that can define both of them at once.